File: lucy.cpp
#include <iostream>
# include<string>
using namespace std;

class myneighbour {
public:
    string name;
    string hobby;
    string status;
    string religion;
};
int main() {
    myneighbour neighbour;

    neighbour.name="stacy kadzo";
    neighbour.hobby="drawing";
    neighbour.status="";
    neighbour.religion="christian";
    
    cout << neighbour.name << " likes " << neighbour.hobby << " drawing " << endl;
    cout <<  neighbour.name << " is " << neighbour.status << " single " <<endl;
    cout << neighbour.name << " is a " << neighbour.religion << endl;

    return 0;
}

File: lucy.py
class MyNeighbour:
    def _init_(self):
        self.name = ""
        self.hobby = ""
        self.status = 0
        self.religion= ""
        

neighbour = MyNeighbour()

neighbour.name = "stacy kadzo"
neighbour.hobby = "drawing"
neighbour.status = "single"
neighbour.religion = "christian"


print(f"{neighbour.name} loves {neighbour.hobby}")
print(f"{neighbour.name} is {neighbour.status}")
print(f"{neighbour.name} is a {neighbour.religion}")
